src/Tasks/SystemState.cs

```diff
@@ -89,11 +84,6 @@ internal sealed class SystemState : StateFileBase, ITranslatable
         /// </summary>
         private GetAssemblyMetadata getAssemblyMetadata;
 
-        /// <summary>
-        /// Cached delegate.
-        /// </summary>
-        private FileExists fileExists;
```

Comment:
What's the motivation for this change?

Reply:
It isn't used, and I came across it. Small cleanup.